lib/ipmi_backend_utils.pm

```diff
@@ -43,39 +43,18 @@ sub switch_from_ssh_to_sol_console {
     save_screenshot;
 }
 
-my $grub_ver;
+my $grub_ver = "grub2";
```

Comment:
quick question, refer to` $grub_ver = "grub2"`;  we do not support grub1 both SLE and openSUSE together now, right?

Reply:
I think there is no older release with grub1 in our test any more, so I remove grub1. Correct me if it is not the truth.

Reply:
yes,  refer to our osd tests, `$grub_ver `should only support `"grub2"` for SLE (e.g 12SP5 and 15+) and openSUSE (LEAP and TW)together. I was not clear if lib/ipmi_backend_utils.pm this test module just only work for us. add a comment, such as we just support grub2 from our osd test as suggestion.

Reply:
Thank you for reminding. I just did some reseach, the function is mostly used called by tests under tests/virt_autoinst. A few  cases it is called in other places have had the ipmi backend and kvm/xen hypervisor preconditions, so I guess it should be used by virt tests only.